MapboxGLAndroidSDKTestApp/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.10)
+
+if (NOT ANDROID_NDK_TOOLCHAIN_INCLUDED)
+    message(FATAL_ERROR "-- Toolchain file not included, see https://developer.android.com/ndk/guides/cmake")
+endif ()
+
+add_library(
+    example-custom-layer MODULE
+    ${PROJECT_SOURCE_DIR}/example_custom_layer.cpp
+)
+
+target_include_directories(
+    example-custom-layer
+    PRIVATE
+        ../../../../MapboxGLAndroidSDK/src/main/cpp
```

Comment:
Do we need all cpp files?

Reply:
we need to point the whole folder as a public C++ API
for now, it contains only a single custom_layer.hpp, but probably could be extended later

Reply:
That's actually my point - if it gets extended we may not want to include it to `MapboxGLAndroidSDKTestApp` automatically. As I understand it 😄

Reply:
It is instructions for CMake to build correct paths to search headers, but the usage of every header is the responsibility of user code in the #include section. Seems other headers will not be included in compilation without explicit reference in user code.
As I see it is the best way to manage dependencies, am I wrong?

Reply:
Yeah, thought again about it - guess it's valid option here.